frontend/src/AlertConfiguration/AlertConfigurationComponents/AlertConfiguration.js

```diff
@@ -25,14 +24,18 @@ export default function AlertConfiguration() {
 
   const [configs, setConfigs] = useState([]);
 
+  fetch("/api/v1/configurations").then(response => {
+    console.log(response);
```

Comment:
do you still want this `console.log`? is this for debugging purposes?

Reply:
I have removed this.